advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>Can you just use the <code>s3_bucket</code> Python variable from earlier? That's part of the value of setting these variables at the top.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Done